bin/run

```diff
@@ -9,17 +9,19 @@ require_relative '../lib/board'
 require_relative '../lib/number_validator'
 require_relative '../lib/outcome_checker'
 require_relative '../lib/player'
+require_relative '../lib/ui'
 
 console = Console.new(stdout: $stdout, stdin: $stdin)
 board = Board.new
 outcome_checker = OutcomeChecker.new(board:)
-display = Display.new
+display = Display.new(console:)
 number_validator = NumberValidator.new
-prompt = Prompt.new(console:, number_validator:,
-                    board:)
+prompter = Prompt.new(console:, number_validator:,
+                      board:)
+ui = UI.new(display:, prompter:)
 player_one = Player.new(marker: 'X')
 player_two = Player.new(marker: 'O')
 players = [player_one, player_two]
-game_looper = GameLooper.new(console:, display:, prompt:, board:, players:, outcome_checker:)
-game = Game.new(console:, game_looper:)
+game_looper = GameLooper.new(ui:, board:, players:, outcome_checker:)
+game = Game.new(ui:, game_looper:)
```

Comment:
**suggestion (non-blocker)**: Does the `Game` need to know about the UI?

An alternative to injecting the collaborator into `Game` might be to use inversion of control by passing in a block. Something like the suggestion below would eliminate the dependency between the `Game` and the `UI`. It might also make you question if `Game` is needed at all. 🤔 

```suggestion
game = Game.new(game_looper:)
ui.with_greeting_and_salutations do
  game.run
end
```

Reply:
Game has definitely been on my mind. I was thinking about keeping it around until I refactor a bit more, with the changes I sketched out in the "brainstorm" diagram on the Miro board, just to see if there are any responsibilities it would end up having outside of starting up the Game Looper. I think it might not end up adding any value, but I don't know if I can predict well enough to make that decision yet. 

As an intermediate step, I'll try passing in the block so it doesn't have to know about UI - thanks!